kogito-springboot-examples/process-postgresql-persistence-springboot/src/main/resources/postgresql.properties

```
@@ -5,11 +5,9 @@ kogito.persistence.type=postgresql
 #Details https://www.postgresql.org/docs/9.6/static/libpq-connect.html#LIBPQ-CONNSTRING
 kogito.persistence.postgresql.connection.uri=postgresql://kogito-user:kogito-pass@localhost:5432/kogito
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/kogito
-spring.datasource.username=kogito-user
-spring.datasource.password=kogito-pass
-#run create tables scripts during the application startup
-spring.flyway.enabled=true
-spring.flyway.baseline-on-migrate=true
-
-
+#run flyway migration during the application startup
+flyway.enabled=true
+flyway.baselineOnMigrate=true
```

```suggestion
spring.flyway.enabled=true
```
I think baseline is not required, also use spring. prefix